### ISSUE TITLE
Hello!! Acceptance rage question!

### ISSUE THREAD

**Issue by scj0709** [SafeAILab/EAGLE#191]
![Image](https://github.com/user-attachments/assets/7c7eec53-f1f4-4a0f-8198-c75dc954c1ef)
Hello!

I was deeply impressed by your paper. I would like to use EAGLE in our SD. 
However, while experimenting with the acceptance rate, I encountered a question.

The model I trained myself shows the following accuracies: 
{'test/0_acc': 0.7011205932523408} 
{'test/1_acc': 0.3695069132237691} 
{'test/2_acc': 0.20591324651566492} 
{'test/3_acc': 0.12545586495531533} 
{'test/4_acc': 0.08118215044928648}

As n increases, the accuracy drops significantly. However, in your paper, the values remain almost consistent as n increases. How is this possible? Even when I use your pre-trained model, I observe a similar drop in accuracy.

Is the accuracy measured in your training code different from n-a?

**Comment by hongyanz** [comment]
n-a is defined as the accept rate when n drafted tokens are used to predict the current position, **conditioned on all previous drafted tokens are accepted.**
